lib/puppetserver/ca/action/generate.rb

```diff
@@ -143,15 +140,25 @@ def generate_pki(settings, signing_digest, subject_alt_names = '')
             [settings[:cakey], int_key],
           ]
 
+          errors = FileSystem.check_for_existing_files(public_files.map { |f| f.first })
```

Comment:
little golf-y but `(&:first)` works too.

Reply:
Oh yeah, always forget about that.